packaging/docker/run.sh

```diff
@@ -63,18 +63,21 @@ done
 export HOME="$GF_PATHS_HOME"
 
 if [ ! -z "${GF_INSTALL_PLUGINS}" ]; then
-  OLDIFS=$IFS
-  IFS=','
-  for plugin in ${GF_INSTALL_PLUGINS}; do
-    IFS=$OLDIFS
-    if [[ $plugin =~ .*\;.* ]]; then
-        pluginUrl=$(echo "$plugin" | cut -d';' -f 1)
-        pluginInstallFolder=$(echo "$plugin" | cut -d';' -f 2)
-        grafana cli --pluginUrl ${pluginUrl} --pluginsDir "${GF_PATHS_PLUGINS}" plugins install "${pluginInstallFolder}"
-    else
-        grafana cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install ${plugin}
-    fi
-  done
+  >&2 echo "\033[0;33mWARN\033[0m: GF_INSTALL_PLUGINS is deprecated. Use GF_PLUGINS_PREINSTALL or GF_PLUGINS_PREINSTALL_SYNC instead. Checkout the documentation for more info."
+  if [ "${GF_INSTALL_PLUGINS_FORCE}" = "true" ]; then
```

Comment:
By including this change we are breaking existing installs, please revert this change.